MACHINE_LEARNING/image_recognizer/image_recognizer_app/views.py

```diff
@@ -10,29 +10,36 @@
 # with Jalasoft.
 #
 
+from django.db.models import base
 from django.views import View
 from django.http import JsonResponse
 from pathlib import Path
 from .model.prediction import Prediction
 from .utils.checker import Checker
 from .utils.unzip import Unzip
-
+from django.http import HttpResponse
+from .utils.imageRecognizer import ImageRecognizer
+import json
+from .exceptions.machine_learning_exception import MachineLearningException
+from .exceptions.error_response import ErrorResponse
 
```

Comment:
Please add an space, to standardize the format in all the files

Reply:
Done!